src/tests/utils/sanitize-url.spec.tsx

```diff
@@ -1,32 +1,91 @@
-import { sanitizeQueryUrl } from "../../app/utils/query-url-sanitization";
+import {
+  isAllAlpha, isDeprecation, sanitizeQueryUrl
+} from '../../app/utils/query-url-sanitization';
+
+
+describe('isAllAlpha should ', () => {
+  const list = [
+    { key: 'aaa', isAllAlphabetic: true },
+    { key: 'aZa', isAllAlphabetic: true },
+    { key: '111', isAllAlphabetic: false },
+    { key: '1a1', isAllAlphabetic: false }
+  ];
+
+  list.forEach(element => {
+    it(`return ${element.isAllAlphabetic} for ${element.key}`, () => {
+      const key = isAllAlpha(element.key);
+      expect(key).toBe(element.isAllAlphabetic);
+    });
+  });
+});
+
+describe('isDepraction should ', () => {
```

Comment:
I just realized that I've been saying deprecated wrong all of these years. `depracate` should be `deprecate`

Reply:
I've added more tests